probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/BackgroundTasks.scala

```diff
@@ -18,16 +19,24 @@ object BackgroundTasks extends IntelliJApi {
     doWait()
   }
 
-  @tailrec
   private def doWait(): Unit = {
-    sleep(5.seconds)
+    val config = ProbeConfig.get()
```

Comment:
We'd like to have an overload of `awaitNone` that is public and takes AwaitIdleParams as parameter. Additionally current awaitIdle endpoint should be extended to accept Option[AwaitIdleParams] under the hood (driver should have 2 overloads (AwaitIdleParams, and 0 arg veriant).

Reply:
we have 3 awaitNone overloads now including one with option that seems a little bit dirty for me - check it out again please and tell me what you think now